api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -35,26 +37,25 @@
       public Cluster apply(org.pmiops.workbench.notebooks.model.Cluster firecloudCluster) {
         Cluster allOfUsCluster = new Cluster();
         allOfUsCluster.setClusterName(firecloudCluster.getClusterName());
-        allOfUsCluster.setGoogleId(firecloudCluster.getGoogleId());
-        allOfUsCluster.setGoogleProject(firecloudCluster.getGoogleProject());
-        allOfUsCluster.setGoogleServiceAccount(firecloudCluster.getGoogleServiceAccount());
-        allOfUsCluster.setGoogleBucket(firecloudCluster.getGoogleBucket());
-        allOfUsCluster.setOperationName(firecloudCluster.getOperationName());
+        allOfUsCluster.setClusterNamespace(firecloudCluster.getGoogleProject());
         allOfUsCluster.setStatus(firecloudCluster.getStatus());
-        allOfUsCluster.setHostIp(firecloudCluster.getHostIp());
         allOfUsCluster.setCreatedDate(firecloudCluster.getCreatedDate());
         allOfUsCluster.setDestroyedDate(firecloudCluster.getDestroyedDate());
         allOfUsCluster.setLabels(firecloudCluster.getLabels());;
         return allOfUsCluster;
       }
     };
 
-  private org.pmiops.workbench.notebooks.model.ClusterRequest toFirecloudClusterRequest(ClusterRequest allOfUsClusterRequest) {
+  private org.pmiops.workbench.notebooks.model.ClusterRequest createFirecloudClusterRequest() {
     org.pmiops.workbench.notebooks.model.ClusterRequest firecloudClusterRequest = new org.pmiops.workbench.notebooks.model.ClusterRequest();
-    firecloudClusterRequest.setBucketPath(allOfUsClusterRequest.getBucketPath());
-    firecloudClusterRequest.setServiceAccount(allOfUsClusterRequest.getServiceAccount());
-    firecloudClusterRequest.setLabels(allOfUsClusterRequest.getLabels());
-    firecloudClusterRequest.setJupyterExtensionUri(allOfUsClusterRequest.getJupyterExtensionUri());
+    // TODO: Use real paths and accounts.
+    firecloudClusterRequest.setBucketPath("");
+    firecloudClusterRequest.setServiceAccount("");
```

Comment:
I think based on this:

https://docs.google.com/document/d/1lcUUX_O2vA9fWwflnhaTpvAlaVqMb47olz4lS1UKwc8/edit?ts=59d5394c

in future we won't have to pass this in?

Reply:
That may very well be correct. Current swagger docs say that we need to, but I'll keep a close eye on it.